refinance/repository/base.py

```diff
@@ -1,59 +1,61 @@
 """Base repository with CRUD methods, suitable for simple objects"""
 
-from typing import Generic, Mapping, Type, TypeVar
+from abc import ABC, abstractmethod
+from typing import Any, Generic, Mapping, Type, TypeVar
 
 from fastapi import Depends
 from sqlalchemy.orm import Query, Session
 
 from refinance.db import get_db
 from refinance.errors.common import NotFoundError
 from refinance.models.base import BaseModel
-from refinance.schemas.base import PaginationSchema
+from refinance.schemas.base import BaseFilterSchema, PaginationSchema
 
-M = TypeVar("M", bound=BaseModel)  # model
-K = TypeVar("K", int, str)  # primary key
```

Comment:
Making typevars private reduces readability, provides no benefit

Reply:
Readability is a moot point. I am personally more accustomed to reading them with underlining. And as it seems to me, the underscore is more common and accepted as a standard in generic.

As for advantages - we forbid them to be imported by other modules and exclude them from the auto-import functionality in IDE. Anything that is not allowed should be forbidden.